.github/workflows/run-standard-tests.yml

```diff
@@ -0,0 +1,194 @@
+            if [[ "${{ matrix.running-mode }}" != "parallel" ]] ; then
+              python $APPLY_PY errors test/LearningTest/TestCoclustering/Standard | tee test/LearningTest/TestCoclustering/Standard/errors.txt
+              python $APPLY_PY errors test/LearningTest/TestKNITransfer/Standard | tee test/LearningTest/TestKNITransfer/Standard/errors.txt
+              python $APPLY_PY errors test/LearningTest/TestKNITransfer/MultiTables SpliceJunction | tee test/LearningTest/TestKNITransfer/MultiTables/errors.txt
+            fi
+          else
+            python3 $APPLY_PY errors test/LearningTest/TestKhiops/Standard IrisLight | tee test/LearningTest/TestKhiops/Standard/errors.txt
+            if [[ "${{ matrix.running-mode }}" != "parallel" ]] ; then
+              python $APPLY_PY errors test/LearningTest/TestCoclustering/Standard Iris | tee test/LearningTest/TestCoclustering/Standard/errors.txt
+              python $APPLY_PY errors test/LearningTest/TestKNITransfer/Standard Iris | tee test/LearningTest/TestKNITransfer/Standard/errors.txt
+            fi
+          fi
+      - name: Check results
+        shell: bash
+        run: |
+          if grep -qr "error" --include="errors.txt" test/LearningTest/ ; then
+            echo "::error::Errors in test"
+            false
+          fi
+          if grep -qr "The test has not been launched" --include="errors.txt" test/LearningTest/ ; then
+            echo "::error::Test not launched"
+            false
+          fi
+      - name: Upload test results
+        if: failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}-${{ matrix.running-mode
+            }}
+          retention-days: 7
+          path: |-
+            test/LearningTest/**/results
+            test/LearningTest/**/results.ref
+            test/LearningTest/**/errors.txt
+            test/LearningTest/**/comparisonResults.log
+  cleanup-cache:
+    # Clean up all caches belonging to the current run
+    if: ${{ always() }}
+    needs: run-standard-tests
+    runs-on: ubuntu-latest
+    permissions:
+      actions: write
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+        with:
+          sparse-checkout: .git
+      - name: Cleanup cache
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |-
+          gh extension install actions/gh-actions-cache
+          REPO=${{ github.repository }}
+          echo "Fetching list of cache key for the current workflow"
+          cacheKeysForDelete=$(gh actions-cache list --key ${{ github.run_id }} | cut -f 1 )
+          set +e
+          echo "Deleting caches..."
+          for cacheKey in $cacheKeysForDelete
+          do
+            gh actions-cache delete $cacheKey --confirm
+          done
+          echo "Done"
```

Comment:
In shell it is more common to use `snake_case` than `camelCase`

Reply:
done (it was a copy/paste from github doc)

Reply:
Ok